appscale/tools/appscale.py

```diff
@@ -239,10 +239,12 @@ def init(self, environment=None):
     shutil.copy(self.TEMPLATE_APPSCALEFILE, appscalefile_location)
 
 
-  def up(self):
+  def up(self, update=[]):
```

Comment:
Default arguments shouldn't be mutable since any change to it will affect the default arg for future calls.